### DISCUSSION TITLE
Remove differentiation between lower- and uppercase letters?

### DISCUSSION THREAD

**Discussion by SoleInvictus** [TandoorRecipes/recipes#1133] [Ideas]
Hello! I love Tandoor Recipes, thank you so much for working on it. I have one request/gripe: the ingredient list currently differentiates between lowercase and uppercase letters. For example, Broccoli and broccoli would be two different ingredients. Because of this, I've unintentionally collected dozens of duplicates over time. Is there any way to change this behavior?

**Comment by smilerz** [comment]
You can merge Broccoli and broccoli on the food management page (/list/food) which will update all occurrences with your preferred spelling.

If you 'Merge and Automate' it will automatically convert the foods when importing new recipes.

![image](https://user-images.githubusercontent.com/1808639/145843809-10cd830e-1f8f-4244-b791-4f19190a778b.png)

**Comment by SoleInvictus** [comment]
That's true and is what I've been doing, but it's a fair bit of extra work
given I have hundreds of ingredients and about 1/3 of them are duplicates.
For example, if I have "Green Onions" and "green onions" automatically
merging, if I import a recipe using "Green onions", I have to set it up yet
again. Removing case sensitivity would avoid all of this.

**Reply by vabene1111** [reply]
i guess some kind of batch edit feature that recommends certain either automations or merges is something that would bee cool. I will create an issue for "batch edit features" so that when we re-work the old one we can add things we collected there.
